web-ui/src/main/resources/catalog/components/viewer/partials/mainviewer.html

```diff
@@ -40,6 +40,14 @@
       <span class="fa fa-print"></span>
       <span role="tooltip" data-translate="">Print</span>
     </button>
+
+    <button class="btn btn-default" type="submit"
+            data-ng-click="syncMod(map)">
+      <span class="fa fa-sync"
+            data-ng-class="synAllLayers ? 'fa-lock' : 'fa-unlock'"/>
+      <span role="tooltip" data-translate="">Sync All Layers</span>
```

Comment:
I'm not 100% sure user will understand the lock icon meaning (BTW I've no better suggestion), but the tooltip could be a bit more detailed "Synchronize all layers (or only background layers) between map viewer and search overview map." maybe